chatsky/conditions/standard.py

```diff
@@ -41,17 +41,20 @@ class ExactMatch(BaseCondition):
     Whether fields set to ``None`` in :py:attr:`.match` should not be compared.
     """
 
-    def __init__(self, match: MessageInitTypes, *, skip_none=True):
-        super().__init__(match=match, skip_none=skip_none)
+    @field_validator("match", mode="before")
+    def validate_match(obj):
```

Comment:
validators should be classmethods: https://docs.pydantic.dev/dev/concepts/validators/#using-the-decorator-pattern